pkg/controller/volume/persistentvolume/index_test.go

```diff
@@ -1524,8 +1524,10 @@ func TestFindMatchVolumeWithNode(t *testing.T) {
 		},
 	}
 
+	enableStorageObjectInUseProtection := utilfeature.DefaultFeatureGate.Enabled(features.StorageObjectInUseProtection)
+
 	for name, scenario := range scenarios {
-		volume, err := pvutil.FindMatchingVolume(scenario.claim, volumes, scenario.node, scenario.excludedVolumes, true)
+		volume, err := pvutil.FindMatchingVolume(scenario.claim, volumes, scenario.node, scenario.excludedVolumes, true, enableStorageObjectInUseProtection)
```

Comment:
Shouldn't this test be in the `k8s.io/component-helpers/storage/volume` package?